TACC/Core-Styles
# fix: monospace inline link needs styles

## Overview

Style monospace text (`<code>`, `<kbd>`, etc.) that are within a link (`<a>`).

## Related

- [unofficial bug report](https://tacc-team.slack.com/archives/CQ1SRUH9P/p1704223582208699)

## Changes

- **added** styles for linked monospace elements

## Testing

1. Open http://localhost:3000/components/detail/monospace--docs.
2. See "A sentence with `code (<code>) snippet`, and as a [link](http://localhost:3000/components/preview/monospace--docs#)."
3. See "A sentence with `code (<code>) with keyboard input`, and as a [link](http://localhost:3000/components/preview/monospace--docs#)."
4. Verify linked monospace text is styled differently than non-link monospace text.
5. Verify link styles change upon interaction (hover, click, focus).

## UI

<img width="960" alt="fix monospace links" src="https://github.com/TACC/Core-Styles/assets/62723358/8e9e84c3-a8f6-4b99-8b50-601cd2c7153c">

https://github.com/TACC/Core-Styles/assets/62723358/b4cd362c-6927-465b-b410-e60bf9b637fb
